rmw_iceoryx2_cxx/src/impl/message/typesupport.cpp

```diff
@@ -6,3 +6,28 @@
 // which is available at https://opensource.org/licenses/MIT.
 //
 // SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#include "rmw_iceoryx2_cxx/impl/message/typesupport.hpp"
+#include "rmw/error_handling.h"
+
+const rosidl_message_type_support_t* get_handle(const rosidl_message_type_support_t* type_support,
+                                                const char* identifier) {
+    const rosidl_message_type_support_t* next = type_support;
+
+    // Keep trying until we run out of type supports to check
+    while (next != nullptr) {
+        // Try to get handle from current type support
+        if (auto ts = get_message_typesupport_handle(next, identifier); ts) {
```

Comment:
nitpick: the comment mentions `current` but the variable is called `next`. What do you think about also using `current` for the variable name?